test/examples/ucp_hello_world.c

```diff
@@ -421,14 +375,12 @@ int main(int argc, char **argv)
     int ret = -1;
 
     /* Parse the command line */
-    if (parse_cmd(argc, argv, &server) != UCS_OK) {
-        goto err;
-    }
+    status = parse_cmd(argc, argv, &server);
+    CHKERR_JUMP(status != UCS_OK, "parse_cmd\n", err);
+
     /* UCP initialization */
     status = ucp_config_read(NULL, NULL, &config);
-    if (status != UCS_OK) {
-        goto err;
-    }
+    CHKERR_JUMP(status != UCS_OK, "ucp_config_read\n", err);
 
     ucp_params.features = UCP_FEATURE_TAG;
```

Comment:
Please use correct initialization procedure for ucp_init() as it described in PR #1297